bower.json

```diff
@@ -22,6 +22,6 @@
     "tests"
   ],
   "dependencies": {
-    "ayepromise": "~1.1.2"
+    "es6-promise": "^3.2.1"
```

Comment:
I had a quick look around and found this library called 'native-promise-only' and it's smaller and seems to work fine 👍